src/NBitcoin/ConsensusOptions.cs

```diff
@@ -56,6 +65,7 @@ public ConsensusOptions()
             this.MaxBlockBaseSize = 1000000;
             this.MaxBlockSigopsCost = 80000;
             this.MaxStandardTxSigopsCost = this.MaxBlockSigopsCost / 5;
+            this.EnforceMinProtocolVersionAtBlockHeight = 0;
```

Comment:
I think we should reconsider this as a consensus option, nothing stops you from hard coding the height in the behaviour itself (or pass it as an argument) this will anyway never be a configuration value.

Reply:
I agree that there are number of different ways in which this could have been achieved. I have decided to add it to the ConsensusOptions so that the behaviour could be configured for any network in case we would need a hard-fork capabilities.

Reply:
This is not a consensus configuration though, we do have other places where we handle network monitoring and we hard code such values in the behaviour.

Also I don't think it should go in the BaseFeature